client/components/MusicWidget.jsx

```diff
@@ -1,8 +1,7 @@
 import React, { useRef, useState } from 'react';
-// import YouTubePlayer from './YoutubePlayer';
 import YoutubeFunctionality from './YoutubePlay';
 
-export default function MusicWidget() {
+export default function MusicWidget(props) {
```

Comment:
You pass `props` but don't use them? Do you plan to? You should probably remove this parameter until you actually use it.